js/index.js

```diff
@@ -22,3 +22,47 @@ const copyright = document.createElement('p')
 copyright.innerHTML = `\u00A9 Michael Castro ${thisYear}`
 footer.appendChild(copyright)
 
+let messageForm = document.querySelector("form[name = 'leave_message']")//selects the form from html
+
+//creates submit form that takes users name, email, & messages.
+messageForm.addEventListener("submit", function(event){
+    event.preventDefault();
+    const form = event.target;//stores the form itself
+
+    let name = form.usersName.value;
+    let usersEmail = form.usersEmail.value;
+    let usersMessage = form.usersMessage.value;
+
+    console.log(form,name, usersEmail)
+
+    let messageSection = document.getElementById("messages");
+    messageSection.classList.toggle("hide-messages-section")
```

Comment:
The line 39 has a bug, try to add more than one message and check your message list, you'll see that when adding every other message, the entire list won't be displayed.